SpatialGDK/Source/SpatialGDK/Private/LoadBalancing/OwnershipLockingPolicy.cpp

```diff
@@ -26,7 +26,7 @@ ActorLockToken UOwnershipLockingPolicy::AcquireLock(AActor* Actor, FString Debug
 {
 	if (!CanAcquireLock(Actor))
 	{
-		UE_LOG(LogOwnershipLockingPolicy, Error, TEXT("Called AcquireLock when CanAcquireLock returned false. Actor: %s."), *GetNameSafe(Actor));
+		UE_LOG(LogOwnershipLockingPolicy, Error, TEXT("Called AcquireLock but CanAcquireLock returned false. Actor: %s."), *GetNameSafe(Actor));
```

Comment:
Other nitpick - CanAcquireLock is a private function that doesn't mean anything to a user. Might want to log why it returned false or log that in the function and just log that it failed here.

Reply:
CanAcquireLock will print an error itself